CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ here on every PR where this is applicable.
 
 ### Fixed
 
+* Set launch container build context to location of override Dockerfile, if provided by @bcsherma in https://github.com/wandb/wandb/pull/7339
```

Comment:
quick note: double check that a user would understand this wording

it might be fine, but for someone without much context about the launch feature (e.g. myself) it is hard to parse. For someone who knows about launch, I'd just double check that the phrase "launch container build context" is familiar, for example if it used in our documentation